app/assets/stylesheets/application/lessons.scss

```diff
@@ -319,3 +255,41 @@
   font-size: 0.9rem;
   text-decoration: none;
 }
+
+
+.section-topics.page-show, .section-lessons.page-show {
+  .lesson-planner-link {
+    float: right;
+  }
+}
+
+.section-topics.page-index .lesson-planner-link {
+  text-transform: uppercase;
+}
+
+form.add_remove_lesson {
+  button[value=destroy] {
+    display: none !important;
+  }
+
+  input[name=_method][value=delete] + .actions {
+    button[value=add] {
+      display: none !important;
+    }
+
+    button[value=destroy] {
+      display: flex !important;
```

Comment:
The spec here is that "Add To Lesson" should change to a "Remove From Lesson" button when clicked (and vice versa). The choice as I see it is to render only one add/remove button and have Javascript update it when the form is submitted so that it shows the opposite text/icon, or render both buttons and have some logic for toggling between which of the two is displayed/hidden. The latter turned out to be simpler when I went to implement it, and I also like that it avoids having Javascript become responsible for the buttons' contents. This keeps the form template in one view on the backend and JS is just responsible for toggling what HTTP verb the form uses (CSS in turn sees this and makes sure the correct submit action is visible). I also thought it would be easier to cache since the form structure doesn't change depending on the current lesson plan state.